.github/workflows/code_quality.yml

```diff
@@ -20,12 +20,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        toxenv: [isort, black, flake8, docs]
+        toxenv: [isort, black, flake8]
+    env:
+      PIP_CONSTRAINT: pip-constraints.txt
```

Comment:
I've seen the Slack thread about this issue here - which library is causing problems? Since OF itself seems to install just fine on CI?

... it could maybe be because of our dependency cache that it's still working :thinking:

Reply:
Hmm I tried it again and it works now, it may have been an issue for python3.10/3.11